proto/elys/tradeshield/tx.proto

```diff
@@ -174,7 +178,7 @@ message MsgCancelPerpetualOrders {
   option (cosmos.msg.v1.signer) = "owner_address";
   option (amino.name) = "tradeshield/MsgCancelPerpetualOrders";
   string owner_address = 1 [ (cosmos_proto.scalar) = "cosmos.AddressString" ];
-  repeated uint64 order_ids = 2;
+  repeated PerpetualOrderPoolKey orders = 2 [ (gogoproto.nullable) = false ];
```

Comment:
we should not do this, will lead to failure of tx deserailization of old txs

Reply:
hmm, any suggestions ? should we deprecate this message itself and create one ?